lib/versioning/gradle/index.ts

```diff
@@ -35,38 +35,42 @@ const getMajor = (version: string): number | null => {
 const getMinor = (version: string): number | null => {
   if (isVersion(version)) {
     const tokens = tokenize(version.replace(regEx(/^v/i), ''));
-    const majorToken = tokens[0];
-    const minorToken = tokens[1];
-    if (
-      majorToken &&
-      majorToken.type === TokenType.Number &&
-      minorToken &&
-      minorToken.type === TokenType.Number
-    ) {
-      return +minorToken.val;
+    if (tokens) {
+      const majorToken = tokens[0];
+      const minorToken = tokens[1];
+      if (
+        majorToken &&
+        majorToken.type === TokenType.Number &&
+        minorToken &&
+        minorToken.type === TokenType.Number
+      ) {
+        return +minorToken.val;
+      }
+      return 0;
     }
-    return 0;
   }
   return null;
 };
 
 const getPatch = (version: string): number | null => {
   if (isVersion(version)) {
     const tokens = tokenize(version.replace(regEx(/^v/i), ''));
-    const majorToken = tokens[0];
-    const minorToken = tokens[1];
-    const patchToken = tokens[2];
-    if (
-      majorToken &&
-      majorToken.type === TokenType.Number &&
-      minorToken &&
-      minorToken.type === TokenType.Number &&
-      patchToken &&
-      patchToken.type === TokenType.Number
-    ) {
-      return +patchToken.val;
+    if (tokens) {
+      const majorToken = tokens[0];
+      const minorToken = tokens[1];
+      const patchToken = tokens[2];
+      if (
+        majorToken &&
+        majorToken.type === TokenType.Number &&
+        minorToken &&
+        minorToken.type === TokenType.Number &&
+        patchToken &&
+        patchToken.type === TokenType.Number
+      ) {
+        return +patchToken.val;
```

Comment:
```suggestion
        return patchToken.val as number;
```

should be fixed fot `getMajor` too please.